docs/architecture.md

```diff
@@ -40,7 +40,7 @@ Gatling Runner Pod runs multiple containers. The diagram below shows how those c
 - gatling-runner
   - Executes Gatling load testing scenario
   - Stores its generated report file (`simulation.log`) to an emptyDir volume
-- gatling-result-transferer
+- gatling-result-transfer
```

Comment:
The original one (`transferer`) should be ok.  We named it`transferer` to mean the one who transfer something 

```suggestion
- gatling-result-transferer
```

Reply:
Thank you. Fixed!
I found `gatling-result-transfer` elsewhere and fixed it.

https://github.com/st-tech/gatling-operator/pull/56/commits/6b54aee130f273f49d4f0c5645ec508c1d1adc33